components/ai_chat/core/browser/conversation_handler.cc

```diff
@@ -833,11 +835,9 @@ void ConversationHandler::GenerateQuestions() {
 }
 
 void ConversationHandler::PerformQuestionGeneration(
-    std::string page_content,
-    bool is_video,
-    std::string invalidation_token) {
+    PageContentses page_contents) {
```

Comment:
The naming is inconsistent, can we just declare it as `PageContents`?

Reply:
In my head I'd called the singular one `PageContents` so this needed to be `PageContentses` - yours is much better!